aten/src/ATen/native/cuda/CuFFTPlanCache.h

```diff
@@ -255,7 +255,13 @@ class CuFFTConfig {
     // Since cuFFT has limited non-unit stride support and various constraints, we
     // use a flag to keep track throughout this function to see if we need to
     // input = input.clone();
+
+#ifdef __HIP_PLATFORM_HCC__
```

Comment:
@jeffdaily , do we still use this preprocessor? Or you'd recommend anything else for hipclang?

Reply:
still do, for now.  will be deprecated soon, but still the one to use.